File: src/main/java/xyz/oribuin/lilori/commands/games/CmdFeed.java
package xyz.oribuin.lilori.commands.games;

import net.dv8tion.jda.api.EmbedBuilder;
import xyz.oribuin.lilori.managers.command.Command;
import xyz.oribuin.lilori.managers.command.CommandEvent;

import java.awt.*;
import java.util.Collections;

public class CmdFeed extends Command {
    private int cookies;

    public CmdFeed() {
        this.name = "Feed";
        this.aliases = Collections.singletonList("Snack");
        this.description = "Feed' Lil Ori Cookies.";
        //this.arguments = "";
    }

    @Override
    public void executeCommand(CommandEvent event) {
        String[] args = event.getMessage().getContentRaw().split(" ");

        if (args.length >= 2) {
            event.reply("**Lil' Ori now has " + cookies + " cookies!**");
            return;
        }

        cookies++;
        EmbedBuilder embedBuilder = new EmbedBuilder()
                .setAuthor("You gave Lil' Ori A Cookie!")
                .setColor(Color.decode("#33539e"))
                .setFooter("Created by Oribuin", "https://imgur.com/ssJcsZg.png")
                .setDescription("**He now has " + cookies + " Cookies**");

        event.reply(embedBuilder.build());
    }
}

File: src/main/kotlin/xyz/oribuin/lilori/commands/games/CmdFeed.kt
package xyz.oribuin.lilori.commands.games

import xyz.oribuin.lilori.LilOri
import xyz.oribuin.lilori.handler.Category
import xyz.oribuin.lilori.handler.Command
import xyz.oribuin.lilori.handler.CommandEvent

class CmdFeed(bot: LilOri) : Command(bot) {
    init {
        name = "Feed"
        category = Category(Category.Type.GAMES)
        aliases = listOf("Snack")
        description = "Feed' Lil Ori Cookies."
        arguments = emptyList()
    }


    override fun executeCommand(event: CommandEvent) {
        // Put command in maintenance mode.
        if (event.author.id != "345406020450779149") {
        event.sendEmbedReply("\uD83D\uDC99 Command in Maintenance!", "Hey! This command is currently being rewritten into a currency system where you'll " +
                "be able to use cookies to buy stuff!")
        return
            }

    }
}